### ISSUE TITLE
Unable to pip install bluerobotics_navigator due to minus '-' in module name

### ISSUE THREAD

**Issue by PeytonBotik** [bluerobotics/navigator-lib#51]
![image](https://github.com/bluerobotics/navigator-lib/assets/77067630/98d24217-c228-4909-97ab-6a6a554aba03)
Hi, I'm trying to install the python package so I can use the navigator-lib repo, but I can't find a way to avoid this error. I'm running Windows Home 11 and saw in another issue that the library can be successfully installed with a Linux distro. However, I would like to avoid putting WSL on my computer since every time I've tried, it's failed and I've had to restore my computer to a backup (maybe time for a new computer soon). Thanks so much for your time, I apologize if this is a quick fix on my end.

**Comment by RaulTrombin** [comment]
Hi @PeytonBotik, are you using a raspberry pi 4 running the windows 11?
Actually python and C++ libraries where tested and developed on Raspbian and the BlueOS.
Somehow the sourcecode was left on version 0.0.0, and you computer is trying to build it from source, which requires rust and maturin, you can try install then, but I still not tested the windows targets yet.

Other possibility is to try the navigator's Rust library directly,
basically you install rust with a simple link:
https://www.rust-lang.org/tools/install

and here is our most recently navigator rust libraries:
https://crates.io/crates/navigator-rs

There is a lot of documentations and examples:
https://docs.bluerobotics.com/navigator-rs/navigator_rs/index.html

**Comment by PeytonBotik** [comment]
Hi @RaulTrombin, thank you for your quick response. I realize now that I made a mistake in how I was attempting to install and use the package. I was trying to do this on my Windows machine instead of directly on my Raspberry Pi. I appreciate your help, and I'll be closing this issue now since the problem was on my end.
